nix/docker.nix

```diff
@@ -12,7 +12,7 @@
 
 # The main contents of the image.
 , exe
-# Short name of the backend, e.g. jormungandr or shelley
+# "", or "-jormungandr" (as suffix to cardano-wallet)
```

Comment:
The backend function argument will still be either jormungandr or shelley, so this comment doesn't need to be changed.

Reply:
Oops, indeed 👍